package.json

```diff
@@ -21,7 +21,7 @@
     "publish-tokens-vue": "lerna publish from-package --yes"
   },
   "devDependencies": {
-    "lerna": "^3.22.1",
+    "lerna": "^4.0.0",
```

Comment:
I assume this is the reason for the 14k lines of changes in the `package-lock.json`; is it really necessary?

Reply:
I got suggested by npm to update lerna... otherwise it shows me 28 vulnerabilities. Should I?

Reply:
Let’s at least do it in a separate commit.

Reply:
I think we should update lerna first (in separate commit) before releasing 2.0.0

Reply:
mostly to reduce npm vulnerabilities and surface attacks (supply chain attack)

Reply:
Eh, I forgot that this repo only allows squash-merge, so requiring separate commits is kinda pointless…